packages/server/lib/util/file.js

```diff
@@ -21,7 +21,9 @@ class File {
 
     this.path = options.path
 
-    this._lockFileDir = path.join(os.tmpdir(), 'cypress')
+    // If multiple user's write to a specific directory is os.tmpdir, permission errors can arrise.
+    // Instead, we make a user specific directory with os.tmpdir.
+    this._lockFileDir = path.join(os.tmpdir(), `cypress-${os.userInfo().uid}`)
```

Comment:
Note: The error described in the comment occurred on my system so I added this to prevent it happening to anyone else in the future